transmit-lib/src/cmr/transmit/echo/tokens.clj

```diff
@@ -57,7 +58,7 @@
                 (get-in parsed [:token_info :user_name])
                 (errors/throw-service-error
                  :unauthorized
-                 (format "Token [%s] has expired" token))))
+                 (format "Token [%s] has expired" (common-util/scrub-token token)))))
```

Comment:
I think the error message should also indicate the token is not what they passed in.

Something along the lines of:
"Token [%s] has expired. Note the token value has been partially redacted."

Reply:
Done.